### DISCUSSION TITLE
Variablize the test name

### DISCUSSION THREAD

**Discussion by daytimothy** [hardinfo2/hardinfo2#60] [Q&A] [answered]
I am having a brain block.  I want to variablize the test name to randomly call tests in a bash script.  I can't figure out how to do it.
i.e.
x="'FPU FFT'"
hardinfo2 -b $x

**Comment by hwspeedy** [comment, marked as answer]
From a bash shell:
```
export B="'FPU FFT'"
echo $B| xargs hardinfo2 -b
```
